packages/server/test/integration/http_requests_spec.js

```diff
@@ -1137,7 +944,7 @@ describe('Routes', () => {
   //       .expect("Content-Type", /application\/json/)
   //       .expect(JSON.parse(json))
 
-  context('GET /__cypress/iframes/*', () => {
+  xcontext('GET /__cypress/iframes/*', () => {
```

Comment:
I deleted the code this is associated with, but going to leave this in case I missed something.